### DISCUSSION TITLE
Is this production ready? if so, it's read me does not explain for production installation

### DISCUSSION THREAD

**Discussion by garima-trivedi** [psycopg/psycopg#957] [General]
Is psycopg3 ready for production? If so, there is no documentation present to talk about installation on production for psycopg3.
Though same is present for psycopg2.
my question is; should we use binary for production like following? 
pip install "psycopg[binary]"
Or should we create package from source distribution of  psycopg3?

**Comment by dvarrazzo** [comment]
Yes it is.

We had no reports of segfaults of the binary extension.
